docs/manual/idlcxx.rst

```diff
@@ -122,33 +120,32 @@ std::string:
 
 The command line options for custom string containers are:
 
-- bounded-string-template
+- ``bounded-string-template``
 
   - template to use for strings instead of std::string
 
   - replaced tags: {BOUND}
```

Comment:
Maybe also make {BOUND} here into ``{BOUND}`` since we do it in the previous sections too.

Reply:
I agree